pkg/graveler/ref/manager.go

```diff
@@ -46,6 +71,18 @@ func NewKVRefManager(executor batch.Batcher, kvStore kv.StoreMessage, db db.Data
 	}
 }
 
+// getRepository returns RepositoryRecord
+func (m *KVManager) getRepository(ctx context.Context, repositoryID graveler.RepositoryID) (*graveler.RepositoryRecord, error) {
```

Comment:
```suggestion
func (m *KVManager) getRepositoryRec(ctx context.Context, repositoryID graveler.RepositoryID) (*graveler.RepositoryRecord, error) {
```
It is quite confiusing